apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -169,6 +179,7 @@ class UnitOverviewTopRow extends React.Component {
                 text={NEXT_BUTTON_TEXT[unitProgress]}
                 size={Button.ButtonSize.large}
                 style={{marginRight: 10}}
+                onClick={e => this.logTryNowButtonClick(unitProgress)}
```

Comment:
Nit: since `e` isn't being used, I think you can do `() => ...` to make that clear.

Reply:
Good point thank you!